examples/rust-sdk/whirlpool_repositioning_bot/package.json

```diff
@@ -4,8 +4,8 @@
   "private": true,
   "scripts": {
     "build": "cargo build",
-    "format": "cargo clippy --fix --allow-dirty --allow-staged && cargo fmt",
-    "lint": "cargo clippy && cargo fmt --check",
+    "format": "cargo +1.86.0 clippy --fix --allow-dirty --allow-staged && cargo +1.86.0 fmt",
+    "lint": "cargo +1.86.0 clippy && cargo +1.86.0 fmt --check",
```

Comment:
would all these version changes be reverted if https://github.com/orca-so/whirlpools/pull/1190 is merged?